### ISSUE TITLE
Error: Undefined class constant 'CACHE_TTL'

### ISSUE THREAD

**Issue by simonspa** [nextcloud/bookmarks#1226]
**Describe the bug**
Since updating to v3.4.0, I'm getting the above error in the logs whenever the background job runs.

**To Reproduce**
1. Update to 3.4.0 or 3.4.1
2. Observe logs
3. See error

**Server (please complete the following information):**
 - OS: Debian Buster
 - HTTP server: nginx 1.14
 - Database: Mysql 10.3
 - PHP version: 7.4
 - Nextcloud version: 19.0.3
 - Bookmarks app version: 3.4.1
 - Nextcloud external user backend: none

**Nextcloud log (nextcloud/data/nextcloud.log)**
```
[core] Error: Error: Undefined class constant 'CACHE_TTL' at <<closure>>

0. /var/www/nextcloud/lib/private/BackgroundJob/Job.php line 62
   OCA\Bookmarks\BackgroundJobs\PreviewsJob->run(null)
1. /var/www/nextcloud/lib/private/BackgroundJob/TimedJob.php line 57
   OC\BackgroundJob\Job->execute(OC\BackgroundJob\JobList {}, OC\Log {})
2. /var/www/nextcloud/cron.php line 126
   OC\BackgroundJob\TimedJob->execute(OC\BackgroundJob\JobList {}, OC\Log {})

at 2020-09-30T18:32:02+02:00
```

**Comment by tang2554** [comment]
I have encountered the same bug.

**Comment by marcelklehr** [comment]
Mmh, how did you update? OCA\Bookmarks\BackgroundJobs\PreviewsJob.php shouldn't exist in the latest version.

**Comment by simonspa** [comment]
Very interestingly, now I'm checking back at my logs and see that it only appeared a couple of hundred times over the course of one day, between 6:00 and 18:00 and then stopped without coming back. Looks like it only happened after updating to 3.4.0 and before updating to 3.4.1 where you probably removed the legacy files including this call?

This was a regular update via the integrated "Apps" from the regular NC App Store.

I think this has been fixed with v3.4.1 and the ticket can be closed. @tang2554 are oyu on the latest version?

**Comment by tang2554** [comment]
@simonspa I have just upgraded to 3.4.1 from appstore, and the issue disappeared.
